src/src/rx_main.cpp

```diff
@@ -51,7 +51,7 @@
 #define SEND_LINK_STATS_TO_FC_INTERVAL 100
 #define DIVERSITY_ANTENNA_INTERVAL 5
 #define DIVERSITY_ANTENNA_RSSI_TRIGGER 5
-#define PACKET_TO_TOCK_SLACK 200 // Desired buffer time between Packet ISR and Tock ISR
+#define PACKET_TO_TOCK_SLACK 280 // Desired buffer time between Packet ISR and Tock ISR
```

Comment:
This increase is to fit 500Hz and because the LR1121 is soooooo slow processing commands.  Knowing what i know now, this should be wrapped in a define for only the LR1121 so that round trip times for the other transceivers is not affected.

Reply:
Probably a good idea to wrap it then